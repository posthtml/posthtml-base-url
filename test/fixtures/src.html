<img src="test.jpg">
<img src="https://example.com/test.jpg">

<video width="250">
  <source src="media/flower.webm" type="video/webm">
  <source src="https://example.tv/media/flower.mp4" type="video/mp4">
  <track default kind="captions" srclang="en" src="media/tracks/friday.vtt">
</video>

<audio src="media/sample.mp3">
  Fallback content
</audio>

<embed type="video/webm" src="media/flower.mp4" width="250" height="200">

<iframe width="300" height="200" src="embed.html"></iframe>

<input type="image" src="image.jpg" alt="">

<script src="javascript.js"></script>
