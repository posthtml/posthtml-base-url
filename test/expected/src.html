<img src="https://example.com/test.jpg">
<img src="https://example.com/test.jpg">

<video width="250">
  <source src="https://example.com/media/flower.webm" type="video/webm">
  <source src="https://example.tv/media/flower.mp4" type="video/mp4">
  <track default="" kind="captions" srclang="en" src="https://example.com/media/tracks/friday.vtt">
</video>

<audio src="https://example.com/media/sample.mp3">
  Fallback content
</audio>

<embed type="video/webm" src="https://example.com/media/flower.mp4" width="250" height="200">

<iframe width="300" height="200" src="https://example.com/embed.html"></iframe>

<input type="image" src="https://example.com/image.jpg" alt="">

<script src="https://example.com/javascript.js"></script>
